imports/plugins/included/product-admin/client/blocks/VariantPricesForm.js

```diff
@@ -82,58 +83,64 @@ const VariantPricesForm = React.forwardRef((props, ref) => {
     <Card className={classes.card} ref={ref}>
       <CardHeader title={i18next.t("productVariant.prices")} />
       <CardContent>
-        <form
-          onSubmit={(event) => {
-            event.preventDefault();
-            submitForm();
-          }}
-        >
-          <Grid container spacing={1}>
-            <Grid item sm={6}>
-              <TextField
-                type="numeric"
-                className={classes.textField}
-                error={hasErrors(["price"])}
-                fullWidth
-                helperText={getFirstErrorMessage(["price"]) || i18next.t("admin.helpText.price")}
-                label={i18next.t("productVariant.price")}
-                placeholder="0.00"
-                {...getInputProps("price", muiOptions)}
-              />
+        { currentVariant && currentVariant.options ?
```

Comment:
I think you might need to verify that `options` is `an array with length > 0` because I'm not able to set the price on a variant that has no options.